web-ext: add page (#2739)
tldr-pages/tldr: pages/common/web-ext.md

# web-ext

> A command line tool for managing web extension development.

> 

Author: I wasn't sure whether it's better to use the npm link, or their GitHub repository:
https://github.com/mozilla/web-ext

npm sounds good to me. 👍

# web-ext

> A command line tool for managing web extension development.
> Homepage: <https://www.npmjs.com/package/web-ext>.

> - Open the current web extension in Firefox:

Reviewer: ```suggestion
- Run the web extension in the current directory in Firefox:
```

`web-ext run`

> - Open a specific directory's web extension in Firefox:

Reviewer: ```suggestion
- Run a web extension from a specific directory in Firefox:
```

Author: This and the previous comment are literally what I had when I originally wrote the page. :yum: I'll apply these changes in the morning. :+1: Thanks.

`web-ext run --source-dir {{path/to/directory}}`

- Display verbose execution output:

`web-ext run --verbose`

> - Open a web extension in Firefox Android:

Reviewer: ```suggestion
- Run a web extension in Firefox Android:
```
